presidio-image-redactor/README.MD

```diff
@@ -100,7 +100,7 @@ curl example:
 ```
```

Comment:
I would move some of the stuff to docs/image-redactor/index.md , keep this file lean like in the other services

Reply:
I think this one is already very lean. 
I will add a Home as the other have.